src/mixins/document-mixin.ts

```diff
@@ -2,11 +2,22 @@ import { AxiosResponse } from 'axios'
 import axios from '@/utils/axios-auth'
 import { Component, Vue } from 'vue-property-decorator'
 import { DocumentUpload } from '@/interfaces'
+import { PdfPageSize } from '@/enums'
+import pdfjsLib from 'pdfjs-dist/build/pdf'
+pdfjsLib.GlobalWorkerOptions.workerSrc = 'https://cdn.jsdelivr.net/npm/pdfjs-dist@2.0.943/build/pdf.worker.min.js'
```

Comment:
Can we store the JS file locally (so we don't use a CDN)? (in /public/js/)

Reply:
will give it a go

Reply:
added file to `/public/js` and updated reference here